tests/test_enable_disable.py

```diff
@@ -0,0 +1,46 @@
+from random import choice
+
+from sphinxlint import main
+
+BULLET = "- "
+
+
+def test_default(capsys):
```

Comment:
Some docstrings/comments to explain what the test is testing would be nice.

Reply:
You're right!